### PR TITLE
Update mountain car example

### PR DESCRIPTION
**Pull request by HoangMHNguyen**

### CODE REVIEW

File: examples/Active Inference Mountain car.ipynb [hunk 1]
```diff
@@ -1,37 +1,25 @@
 {
```
Comment (bvdmitri) [inline examples/Active Inference Mountain car.ipynb:1]: <p>As a reader, I would not really understand this equation. It is called "the gravitational force" while, in fact, I'm pretty much sure it has little to do with Newton's laws. It seems to be that this is equation is a combination of the landscape and associated forces. As a reader I also would not understand what is the argument "\phi" of the function. Please add more explanation of this equation.</p><p> </p><p>In the <code>create_physics</code> function we also have two extra forces, which are not explained much. We could add some brief explanation to the friction force and to the engine force functions as well.  Should not be very detailed, just a brief explanation would be enough.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/RxInfer.jl/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (bvdmitri) [reply examples/Active Inference Mountain car.ipynb:1]: @Sepideh-Adamiat @HoangMHNguyen

**Comment by HoangMHNguyen** [comment]
Thanks a lot for the feedback @bartvanerp @bvdmitri ! @Sepideh-Adamiat and I will address those issues

**Comment by Sepideh-Adamiat** [comment]
@bvdmitri @bartvanerp Thanks for your comments. We changed the notebook accordingly and made a new PR.

**Review by bartvanerp — APPROVED** [review]
Thanks a lot for the changes! The notebook is now in a much better shape. Great work!

Some small questions:
- In equation (1), should there be some kind of discretisation term, e.g. Δt?
- In the full generative model I changed the starting index of the product to t (instead of k). Please let me know whether this is correct.

@bvdmitri ready for merge imo. I did not have the opportunity to run `make examples` to see whether everything renders correctly.

**Comment by HoangMHNguyen** [comment]
Thanks a lot @bartvanerp !
- It's correct to change the starting index of the product to t. 
- Equation (1) is copied from Thijs's paper and I think it's clear. So there's no need to change it.

**Comment by bvdmitri** [comment]
Thanks guys! I'll take a look on why CI fails.

**Pull request merged**